packages/core/src/typeschema/crawler.ts

```diff
@@ -263,7 +263,7 @@ class AsyncCrawler {
       await this.visitor.onEnterObject(path, obj, schema);
     }
 
-    if (this.excludeMissingProperties) {
+    if (this.excludeMissingProperties && obj.value) {
```

Comment:
Does this account for falsy values (0, false)?

Reply:
Ah I see, it can only be an object. `TypedValue.value` typing being `any` was throwing me off